frontend/e2e/job.spec.js

```diff
@@ -0,0 +1,70 @@
+// @ts-check
+import { test, expect } from '@playwright/test'
+
+async function setup(page) {
+  await page.goto('http://localhost:8000/')
+  // wait 5 seconds for the page to load
+  await page.waitForTimeout(1000)
```

Comment:
👀

Reply:
(This is just the same comment/note as on the Account Acceptance Test PR about 5 seconds vs 1 second)